### PR TITLE
Us029 add get eq cis for ce

### PR DESCRIPTION
**Pull request by tej99**

### CODE REVIEW

**Comment by tej99** [comment]
[US029: INT Select/Load EQ Collection Instruments [XXL]](https://trello.com/c/QharabAR/42-us029-int-select-load-eq-collection-instruments-xxl)

File: test/resources/test_collection_exercise.py [hunk 1]
```diff
@@ -115,7 +139,8 @@ def test_single_collection_exercise(self, mock_request):
                          'Business Register and Employment Survey')
         self.assertEqual(response_data['collection_exercise']['name'], '000000')
         self.assertEqual(response_data['events'][0]['tag'], "mps")
-        self.assertEqual(response_data['collection_exercise']['name'], test_period)
+        self.assertEqual(response_data['events'][0]['tag'], "mps")
```
Comment (benjefferies) [inline test/resources/test_collection_exercise.py]: Duplicate assertion

File: ras_backstage/resources/collection_exercise/single_collection_exercise.py [hunk 2]
```diff
@@ -43,6 +47,7 @@ def get(short_name, period):
             "collection_exercise": full_exercise,
             "events": exercise_events,
             "collection_instruments": collection_instruments,
+            "eq_ci_selectors": eq_ci_selectors,
```
Comment (benjefferies) [inline ras_backstage/resources/collection_exercise/single_collection_exercise.py]: Are we missing a test to check the eq_ci_selectors?

Comment (tej99) [reply ras_backstage/resources/collection_exercise/single_collection_exercise.py]: I've just added an assert in the test_single_collection_exercise to check that eq_ci_selectors are being passed back in the response.

**Pull request merged**